### PR TITLE
7693 common tables

### PR DESCRIPTION
**Pull request by cborla**
|Related issue|
|---|
|#7693|

## Description

The following tables were created and populated:

- Aliases
- Contributors
- Platforms
- References

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
- [ ] Source installation

### CODE REVIEW

File: tools/mitre/mitredb.py [hunk 1]
```diff
@@ -240,6 +251,79 @@ class Mitigations(Base):
...
     deprecated = Column(const.DEPRECATED_t, Boolean, default=False)
 
+    references = relationship(const.REFERENCES_r, backref=const.MITIGATIONS_r)
+
+
+class Aliases(Base):
+    """
+    In this table are stored the aliases of json file
+    The information stored:
+        id: Used to identify the technique, mitigation, group or software (FK) (PK)
```
[the remark this answers is not in the document]
Comment (cborla) [reply tools/mitre/mitredb.py]: Done, also I updated some "Used to identify" references.

File: tools/mitre/mitredb.py [hunk 2]
```diff
@@ -242,6 +254,80 @@ class Mitigations(Base):
...
+    """
+    __tablename__ = "reference"
+
+    Id = Column(const.ID_t, String, primary_key=True)
+    source = Column(const.SOURCE_t, String, nullable=False)
+    external_id = Column(const.EXTERNAL_ID_t, String, default=None)
+    url = Column(const.URL_t, String, default=None)
+    description = Column(const.DESCRIPTION_t, String, default=None)
+
+    def __init__(self, Id="", source="", external_id=None, url=None, description=None) :
```
Comment (TomasTurina) [inline tools/mitre/mitredb.py:326]: Remove __init__ functions

File: tools/mitre/mitredb.py [hunk 3]
```diff
@@ -242,6 +254,80 @@ class Mitigations(Base):
...
+
+
+class Aliases(Base):
+    """
+    In this table are stored the aliases of json file
+    The information stored:
+        id: Used to identify the group or software (PK).
+        alias: Alias related to this item (PK).
+    """
+    __tablename__ = "aliase"
```
Comment (TomasTurina) [inline tools/mitre/mitredb.py:267]: Typo: `alias`

File: tools/mitre/mitredb.py [hunk 4]
```diff
@@ -344,6 +433,31 @@ def parse_table_(function, data_object):
         if const.DEPRECATED_j in data_object:
             table.deprecated = data_object[const.DEPRECATED_j]
 
+    # Alias
+    if data_object.get(const.ALIAS_j):
```
Comment (TomasTurina) [inline tools/mitre/mitredb.py:437]: Create a new function with all this code to not repeat it twice

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: tools/mitre/mitredb.py (hunk 2 above)
Comment (cborla) [reply tools/mitre/mitredb.py]: Done

File: tools/mitre/mitredb.py (hunk 3 above)
Comment (cborla) [reply tools/mitre/mitredb.py]: Fixed

File: tools/mitre/mitredb.py (hunk 4 above)
Comment (cborla) [reply tools/mitre/mitredb.py]: Done

**Pull request merged**